### DISCUSSION TITLE
Hide previous races on schedule

### DISCUSSION THREAD

**Discussion by oOStroudyOo** [marcokreeft87/formulaone-card#237] [Ideas]
Hello,

First of all, this app is amazing, thank you so much for your hard work.

I was just wondering if it was possible to have a true/false option in the Schedule config so we can hide the races that have finished and just leave the remaining races shown?

Thank you.

**Comment by marcokreeft87** [comment]
Yes there is!
Check : https://github.com/marcokreeft87/formulaone-card#configuration and search for previous_race
